cli/command/service/opts.go

```diff
@@ -644,8 +672,14 @@ const (
 	flagName                  = "name"
 	flagNetwork               = "network"
 	flagPublish               = "publish"
+	flagPublishIngress        = "publish-ingress"
+	flagPublishHost           = "publish-host"
 	flagPublishRemove         = "publish-rm"
+	flagPublishRemoveIngress  = "publish-rm-ingress"
+	flagPublishRemoveHost     = "publish-rm-host"
```

Comment:
These should be `publish-ingress-rm`, `publish-host-rm`, same with the add flags